x-pack/plugin/sql/qa/server/security/src/test/java/org/elasticsearch/xpack/sql/qa/security/SqlSecurityTestCase.java

```diff
@@ -56,6 +56,13 @@
 import static org.hamcrest.Matchers.is;
 
 public abstract class SqlSecurityTestCase extends ESRestTestCase {
+    public String elasticsearchAddress() {
+        // CLI only supports a single node at a time so we just give it one.
+        // String cluster = System.getProperty("tests.rest.cluster");
+        // return cluster.split(",")[0];
```

Comment:
Leftover?

Reply:
Ah, I was so sure I had this already removed. Thanks, removed!